### ISSUE TITLE
Make sure elements support ARIA attributes

### ISSUE THREAD

**Issue by jessicaul** [metabase/metabase#53270]
**Is your feature request related to a problem? Please describe.**
To make Metabase more accessible by having elements use only ARIA-supported attributes. 

More information: https://dequeuniversity.com/rules/axe/4.10/aria-allowed-attr?application=AxeChrome

aria-expanded="false" appears in multiple elements like
- New button
- settings button
- view-only label 
- ... menu on a question 
- column name headers 
-  cloud icon for downloads

The aria-expanded property is currently on the button, but they believe it should be on the div. 

**How important is this feature to you?**
Customer request 

Related: https://github.com/metabase/metabase/issues/1462
